docs/options.md

```diff
@@ -45,9 +45,19 @@ replacement
 ```ts
 {
   path: 'lazy',
-  loadChildren: () => System.import('./lazy/lazy.module').then(function(module) {
-    return module['LazyModule'];
-  })
+  loadChildren: () => System.import('./lazy/lazy.module').then(module => module['LazyModule'])
+}
+```
+
+If you prefer to use `dynamic import`, set the `loader` to `import`
```

Comment:
Remove `If you prefer`

Reply:
I add `If you prefer` here because it is also mentioned in `If you prefer to use System.import, set the loader to system`, so should I remove that also?